alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -471,8 +472,27 @@ void RunningPlan::deactivate()
     } else {
         _ae->getTeamObserver().notifyRobotLeftPlan(_activeTriple.abstractPlan);
     }
+
     revokeAllConstraints();
     deactivateChildren();
+
+    /*
+     * Create instance of scheduler, later use shared instance for actual scheduling.
+     */
+    scheduler::Scheduler scheduler;
+
+    _terminateJob = make_shared<scheduler::Job>();
```

Comment:
as a good practice, you should use the namespace along with method calls: std::make_shared